alerts/redis/evicted-keys.json

```diff
@@ -0,0 +1,36 @@
+{
+    "displayName": "Redis - Evicted Keys",
+    "documentation": {
+        "content": "Under memory pressure, the system will evict keys to free up memory. This is an indicator of memory pressure of the redis configuration. This alert fires if at least 5 keys are evicted.",
+        "mimeType": "text/markdown"
+    },
+    "userLabels": {},
+    "conditions": [
+        {
+            "displayName": "Redis - Evicted Keys",
+            "conditionThreshold": {
+                "filter": "resource.type = \"gce_instance\" AND metric.type = \"workload.googleapis.com/redis.keys.evicted\"",
+                "aggregations": [
+                    {
+                        "alignmentPeriod": "300s",
+                        "crossSeriesReducer": "REDUCE_NONE",
+                        "perSeriesAligner": "ALIGN_MEAN"
+                    }
+                ],
+                "comparison": "COMPARISON_GT",
+                "duration": "30s",
```

Comment:
condition_threshold.duration had an invalid value of \"30s\": only multiples of 1 minute are supported